### PR TITLE
Add history model

### PR DESCRIPTION
**Pull request by idealasgas**

### CODE REVIEW

File: app/models/rent_item.rb [hunk 1]
```diff
@@ -8,7 +10,16 @@ class RentItem < ApplicationRecord
...
+
   def tenant_attributes=(attributes)
     self.tenant = Contact.create(attributes)
   end
+
+  private
+
+  def create_history
+    history = ItemHistory.new(item: item, rent_item_id: id)
+    history.save if item.valid?
```
Comment (reedzyy) [inline app/models/rent_item.rb]: Rewrite this method with oneline style

**Review by reedzyy — CHANGES_REQUESTED** [review]

**Review by simonovich — APPROVED** [review]

**Review by reedzyy — APPROVED** [review]

File: app/models/item_history.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class ItemHistory < ApplicationRecord
+  belongs_to :item
+  validates :item, presence: true
+
+  def find
```
Comment (anatoliliotych) [inline app/models/item_history.rb]: rename

**Review by anatoliliotych — CHANGES_REQUESTED** [review]

File: app/views/items/_history.html.slim [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        = history.rent_item.tenant.name
+        - if history.rent_item.tenant.email
+          br/
+          = history.rent_item.tenant.email
+    - else
+      td
+        = history.rent_item.tenant.first_name
+        br/
+        = history.rent_item.tenant.email
+    td = history.rent_item.created_at.strftime '%e %b %Y %k:%M'
```
Comment (anatoliliotych) [inline app/views/items/_history.html.slim]: should be in model/helper not here

Comment (anatoliliotych) [reply app/views/items/_history.html.slim]: I mean formating

File: db/migrate/20180821125245_create_versions.rb [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+  def change
+    create_table :versions do |t|
+      t.string   :item_type, {:null=>false}
+      t.integer  :item_id,   null: false
+      t.string   :event,     null: false
+      t.string   :whodunnit
+      t.text     :object, limit: TEXT_BYTES
+
+      # Known issue in MySQL: fractional second precision
```
Comment (anatoliliotych) [inline db/migrate/20180821125245_create_versions.rb]: do we use mysql? ) remove this comment

**Review by anatoliliotych — CHANGES_REQUESTED** [review]

**Review by anatoliliotych — APPROVED** [review]

**Pull request merged**